CMakeLists.txt

```diff
@@ -88,6 +88,39 @@ include(gmp)
 include(lagrange)
 lagrange_include_modules(bvh)
 
+if(TARGET nlohmann_json::nlohmann_json)
```

Comment:
Please add this to a separate `nlohmann_json.cmake` file to keep the main code clean 😅

Reply:
done